_test_/user-controllers.spec.js

```diff
@@ -0,0 +1,87 @@
+const request = require('supertest');
+const db = require('../data/dbConfig');
+const server = require('../index');
+// const generateToken = require('../src/utils/generate-token');
+
+
+// beforeAll((done) => {
+//   token = generateToken('user');
+//   done();
+// })
```

Comment:
Hey Funmi great work with the tests, all are passing! Let's try to avoid having commented out code in our next Pull Requests. I will merge this PR now.